packages/mesos-modules/windows.build.ps1

```diff
@@ -70,8 +72,12 @@ function Build-Mesos-Modules {
                 "-DBUILD_TESTING=OFF"
             ) `
             -NoNewWindow `
-            -Wait `
             -PassThru
+
+        # `Start-Process -Wait` is prone to hanging when waiting for cmake.
+        # Instead, we use `Wait-Process` here and below.
+        Wait-Process -InputObject $p
```

Comment:
Perhaps a link to the ticket with context, and maybe mention explicitly that we don't know why? (Rather than have the reader wonder if we knew why but didn't explain here)